azurelinuxagent/common/conf.py

```diff
@@ -87,7 +87,7 @@ def load_conf_from_file(conf_file_path, conf=__conf__):
 
 __SWITCH_OPTIONS__ = {
     "OS.AllowHTTP" : False,
-    "OS.EnableFirewall" : False,
+    "OS.EnableFirewall" : True,
```

Comment:
Did you mean to change this?

Reply:
No.  I fixed it to fix the tests, but that was the wrong fix.  Let me make the correct fix.